File: 1920-build-array-from-permutation/1920-build-array-from-permutation.py
class Solution:
    def buildArray(self, nums: List[int]) -> List[int]:
        ans = []

        # for문 돌며 주어진 조건에 맞는 수를 ans에 추가
        for i in range(len(nums)):
            ans.append(nums[nums[i]])

        return ans

File: 1920-build-array-from-permutation/1920-build-array-from-permutation.c
/**
 * Note: The returned array must be malloced, assume caller calls free().
 */
int* buildArray(int* nums, int numsSize, int* returnSize) {
    int *ans = (int *)malloc(sizeof(int) * numsSize);

    // for문 돌며 주어진 조건에 맞는 수를 ans에 추가
    for (int i=0; i<numsSize; i++) {
        ans[i] = nums[nums[i]];
    }

    *returnSize = numsSize;
    return ans;
}